### ISSUE TITLE
The default time limit per attempt

### ISSUE THREAD

**Issue by cubizh** [thewca/wca-regulations#99]
Current regulation states:

```
A1a1) The default time limit per attempt is 10 minutes, though the organisation team may announce a higher or lower time limit.
```

I think the intended spirit of this regulation is to interpret the default time limit as "solve" time instead of "attempt" time, as the attempt timing can be quite hard to measure, given that it includes inspection.

**Comment by lgarron** [comment]
Did you actually read the style guide? I didn't think anyone else ever did. :-P

Fixed in 5691773895a626edcc389847b8d0cb0545115ec0. Thanks. :-)

**Comment by cubizh** [comment]
Yes, the style guide should be a mandatory read by everyone that is doing a translation because certain things are not so common sense to translate/understand the intended goal for regulations/guidelines. The distinction between attempt and solve is indeed one of those cases.
